docs/site/hasOne-relation.md

```diff
@@ -0,0 +1,259 @@
+---
+lang: en
+title: 'hasOne Relation'
+keywords: LoopBack 4.0, LoopBack 4
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/hasOne-relation.html
+---
+
+## Overview
+
+A `hasOne` relation denotes a one-to-one connection of a model to another model
+through referential integrity. The referential integrity is enforced by a
```

Comment:
Based on our recent discussions, the initial implementation is not going to enforce referential integrity. I think it's important to clearly communicate this limitation.

Reply:
Okay, I'll add a sentence in this section as well to let our users know (I go in more detail later in the page).